### PR TITLE
989 namespace drivetrain configurations

### PR DESCRIPTION
**Pull request by Ezward**
Addresses Issue https://github.com/autorope/donkeycar/issues/989

The PWM_STEERING_PIN configuration values are used in several DRIVETRAIN_TYPE configurations, each with a different default value. If you just set the DRIVETRAIN_TYPE = "PWM_STEERING_THROTTLE", but don't uncomment the PWM_STEERING_PIN value, then the default of a different drivetrain get's used.

So a workaround is to just uncomment PWM_STEERING_PIN in the PWM_STEERING_THROTTLE sections.

This fix changes the newer drivetrain configuration into python dictionaries so all configuration are namespaced.

### CODE REVIEW

**Comment by Ezward** [comment]
This branch still needs have all the updated drivetrains tested.

File: donkeycar/templates/cfg_complete.py [hunk 1]
```diff
@@ -177,14 +186,40 @@
...
+# - the pwm pins produce a duty cycle from 0 (completely LOW)
+#   to 1 (100% completely high), which is proportional to the
+#   amount of power delivered to the motor.
+# - in forward mode, the reverse pwm is 0 duty_cycle,
+#   in backward mode, the forward pwm is 0 duty cycle.
+# - both pwms are 0 duty cycle (LOW) to 'detach' motor and
+#   and glide to a stop.
+# - both pwms are full duty cycle (100% HIGH) to brake
+#
+HBRIDGE_PIN_FWD = 18,       ß# provides forward duty cycle to motor
```
Comment (DocGarbanzo) [inline donkeycar/templates/cfg_complete.py:218]: Here is an unintentional `beta` as well as some unintentional commas here and below.

Comment (Ezward) [reply donkeycar/templates/cfg_complete.py]: Thank you for catching that.

**Review by DocGarbanzo — APPROVED** [review]
Looks good to me.

**Review by DocGarbanzo — APPROVED** [review]
LGTM - thank you.

**Pull request merged**